web/static/js/components/css_modules/idea_submission_form.css

```diff
@@ -22,3 +22,7 @@
   top: -3em;
   left: 33%;
 }
+
+#required:after {
+  display: inline;
```

Comment:
ahh, so that's all it took. i'm guessing the :after value was `display: block;`?

Reply:
it was 'display: inline-block;'